src/bitcoin.ts

```diff
@@ -71,40 +105,35 @@ app.use((ctx, _next) => {
 const port = process.env.SIDETREE_BITCOIN_PORT || config.port;
 
 // initialize the blockchain service and kick-off background tasks
+let server: any;
 let blockchainService: SidetreeBitcoinProcessor;
 try {
   blockchainService = new SidetreeBitcoinProcessor(config);
 
-  blockchainService.initialize()
+  // SIDETREE_TEST_MODE enables unit testing of this file by bypassing blockchain service initialization.
+  if (process.env.SIDETREE_TEST_MODE === 'true') {
+    server = app.listen(port);
```

Comment:
shouldn't you still initialize() ?

Reply:
No, purposely not going through the initialization.